src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,75 @@
+package view;
+
+import domain.gamer.Name;
+import dto.AllPlayerResultDto;
+import dto.DealerResultDto;
+import dto.GameResultDto;
+import dto.InitCardDto;
+import dto.PlayerCardStateDto;
+import java.util.List;
+
+public class OutputView {
+
+    private static final int INIT_CARD_COUNT = 2;
+    private static final int MIN_DEALER_SCORE = 16;
+    private static final String NAME_FORMAT_SYMBOL = ", ";
+
+    public void printInitCard(InitCardDto initCardDto) {
+        String playersNames = changeNameFormat(initCardDto.playerNames());
+        System.out.println("\n딜러와 " + playersNames + "에게 " + INIT_CARD_COUNT + "장을 나누었습니다.");
+        System.out.println("딜러: " + changeFormat(initCardDto.dealerCard()));
+
+        for (int i = 0; i < initCardDto.playersCard().size(); i++) {
+            System.out.println(
+                    initCardDto.playerNames().get(i) + "카드: " + initCardDto.playersCard().get(i));
+        }
```

Comment:
인라인 하지 않는게 가독성이 좋을것 같네요.

```suggestion
String playerName = initCardDto.playerNames().get(i);
String playerCard = initCardDto.playersCard().get(i);
System.out.println(playerName + "카드: " + playerCard);
```

Reply:
이렇게 풀어놓고 보니까 뭔가 중복된 느낌이 드네요. interate하는 기준이 playersCard인가요 playersName인가요?

이런식으로 할수 있지 않나요?

static factory가 혹시 인스턴스 변수들을 가리고 있지 않았나요?

```java
public record PlayerCardDto(List<PlayerCardDto> playerCardDtos)

public record PlayerCardDto(String name, String cardStatus)
...

playerCardDtos
        .map(dto -> dto.getName() + "카드: " + dto.getCardStatus())
        .forEach(System.out::println);

```

Reply:
안돌의 코드는 한 플레이어의 이름과 카드 정보를 가진 dto를 만들고 그것을 List로 가지는 dto를 만든 것 같습니다.

저는 List<Name> 과 List<CardStatus> 를 가진 Dto를 만들었구요. iterate하는 기준은 무엇이든 상관없다고 생각했습니다. 사이즈가 똑같다고 생각해서요. 

안돌이 예시로 써주신 코드가 가독성이 좋아 놀랐습니다. 
한편으로는 Dto 하나로 할 수 있는 것을 두개로 쪼개서 하는 것은 괜찮은가? 라는 의문도 들었습니다.
안돌의 생각이 궁금합니다.

Reply:
> iterate하는 기준은 무엇이든 상관없다

가 iterate하는 기준이 애매하다 와 같은 말 아닌가요?

하나로 잘 할수 있으면 문제가 없겠죠? 하나로 **잘 하는** 게 무엇이고 그 조건이 무엇인지 생각해보세요